.prettierignore

```diff
@@ -0,0 +1 @@
+client
```

Comment:
why is .gitignore removed?

Reply:
The gitignore file was created when creating the repo by mistake, so it is removed now.

Reply:
Don't we still need a gitignore file?

Reply:
We can add according to need in that time. As for frontend and backend, they could have their gitignores in their folders since entirely different languages will be used.